### PR TITLE
Patch numpy.array_equal if numpy version > 1.8

### PR DESCRIPTION
**Pull request by AntoineDupre**
See issue #3 

As centos7 default numpy version is 1.7.1, it is maybe better to patch array_equal instead of forcing numpy version.

Tested in a Conda environment with numpy 1.7.1.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,6 +45,9 @@ def get_readme(name='README.rst'):
 
     # Requirements
     install_requires=['pytango>=9.2.1'],
-    tests_require=['pytest-mock', 'pytest-xdist', 'pytest-coverage'],
+    tests_require=['pytest-mock',
+                   'pytest-xdist',
+                   'pytest-coverage',
+                   'numpy>=1.0.8'],
```
Comment (vxgmichel) [inline setup.py]: We need numpy in `install_requires` as well, right?

Comment (vxgmichel) [inline setup.py]: Also: `1.8.0` instead of `1.0.8`

Comment (AntoineDupre) [inline setup.py]: It is already a PyTango requirement.
We keep compatibility with numpy verion under 1.8.0 by patching array_equal method (as 1.7.1 is Centos7 default numpy version).
However, unittest requires numpy above 1.8.0 (to test compare_triplet without the patch)

File: facadedevice/graph.py [hunk 2]
```diff
@@ -1,14 +1,36 @@
-"""Provide objects for building reactive graphs."""
-
```
Comment (vxgmichel) [inline facadedevice/graph.py]: No docstring?

**Comment by vxgmichel** [comment]
Still a WIP?

**Comment by vxgmichel** [comment]
LGTM!

**Comment by vxgmichel** [comment]
Thanks!

**Pull request merged**